pricegraph/bench/pricegraph.rs

```diff
@@ -14,14 +14,19 @@ pub fn read(c: &mut Criterion) {
 pub fn transitive_orderbook(c: &mut Criterion) {
     let pricegraph = read_default_pricegraph();
     let dai_weth = Market { base: 7, quote: 1 };
+    let hops = &[None, Some(1), Some(2), Some(5), Some(10), Some(30)];
 
-    c.bench_with_input(
-        BenchmarkId::new("Pricegraph::transitive_orderbook", *DEFAULT_BATCH_ID),
-        &(&pricegraph, dai_weth),
-        |b, &(pricegraph, dai_weth)| {
-            b.iter(|| pricegraph.transitive_orderbook(dai_weth, None, None))
-        },
-    );
+    let mut group = c.benchmark_group("Pricegraph::transitive_orderbook_with_hops");
```

Comment:
nit: I would leave this the name of the function:
```suggestion
    let mut group = c.benchmark_group("Pricegraph::transitive_orderbook");
```